databuilder/publisher/neo4j_csv_publisher.py

```diff
@@ -381,6 +381,8 @@ def _create_props_body(self,
             # escape quote for Cypher query
             # if isinstance(str, v):
             v = v.replace('\'', "\\'")
+            # escape backslash for Cypher query
+            v = v.replace("\\", "\\\\")
```

Comment:
I think it is ok, but will let @jinhyukchang comment whether if it is safe to do this...

Also should it just be 

            # escape backslash for Cypher query
            v = v.replace("\", "\\\\") ? (combined L384 and L385)

Reply:
I think it actually needs to be `.replace("\\", "\\")`
```python
>>> "\_".replace('\', "\\")
>>> "\_".replace("\", "\\\\")
  File "<stdin>", line 1
    "\_".replace("\", "\\\\")
                            ^
SyntaxError: unexpected character after line continuation character
>>> "\_".replace("\\", "\\\\")
'\\\\_'
>>> "\_".replace("\\", "\\")
'\\_'
```

Reply:
> ```python
> "\_".replace("\\", "\\")
> ```

Interesting. It seems like the code is trying to replace a backslash with a backslash, but it apparently works. 😄